common.eai

```diff
@@ -13306,7 +13291,6 @@ function universal_attack_sequence takes nothing returns nothing
   else
     set br_rush = false
   endif
-  set most_threatened_town = Max(most_threatened_town,0)
```

Comment:
Probrably more dangerous to remove this.

Reply:
Each usage location has a separate judgment -1, which should not be necessary